Cargo.toml

```diff
@@ -450,6 +450,7 @@ sources-logs = [
   "sources-datadog_agent",
   "sources-demo_logs",
   "sources-docker_logs",
+  "sources-dnstap",
```

Comment:
I think `sources-dnstap` is not included in this list because it is only included for *nix platforms. I think adding it here will cause the Windows build to try to include it too. This is, admittedly, confusing so I'm open to better ways to model these components that are conditionally included by platform.